tox.ini

```diff
@@ -7,6 +7,17 @@ envlist =
     readme
 skipsdist = True
 
+[travis]
+python =
+    2.7: py27
+    3.5: py35
+    3.6: py36,flake8,readme
+
+[travis:env]
+DJANGO =
+    1.11: dj111
+    2.0: dj20
```

Comment:
travis and travis:env sections come from:
https://tox-travis.readthedocs.io/en/stable/envlist.html